.github/renovate.json5

```
@@ -360,10 +360,21 @@
 // PR group for Kubernetes CSI
       "groupName": "kubernetes CSI",
       "matchPackagePrefixes": [
-        "kubernetes-csi"
+        "kubernetes-csi",
+        "rook",
       ],
       "separateMajorMinor": "false",
       "pinDigests": false
-    }
+    },
+    {
+// PR group for all the operator framework related things
+      "groupName": "operator framework",
+      "matchPackagePrefixes": [
+        "operator-framework",
+        "redhat-openshift-ecosystem",
+      ],
+      "separateMajorMinor": "false",
+      "pinDigests": false
+    },
```

😬 did you  run `renovate-config-validator`?
```suggestion
    }
```

---

Ups!

---

You can validate it in ci, see: https://github.com/crossplane/crossplane/blob/master/.github/workflows/renovate.yml#L31

---

Uhmm I'll add another PR adding the validator, agree @phisco  ?

---

@phisco now there's a linter please proceed =D